src/app/(guest)/login/page.tsx

```diff
@@ -1,32 +1,36 @@
 'use client';
 
+import React from 'react';
+import { Formik, Form } from 'formik';
+import * as yup from 'yup';
 import Button from '@/components/Button';
 import Input from '@/components/Input';
 import { useAuth } from '@/hooks/auth';
-import { useEffect, useState } from 'react';
 
 const Page = () => {
-  const [errors, setErrors] = useState([]);
+  const { login } = useAuth({ middleware: 'guest' });
 
-  const { login, isLoading, user } = useAuth({ middleware: 'guest' });
+  const submitForm = async (values: any) => {
+    const { email, password } = values;
+    login({ email, password });
+  };
 
-  useEffect(() => {
-    console.log('user....', user);
-    console.log('errors....', errors);
-  }, [user, errors]);
+  const validationSchema = yup.object().shape({
+    email: yup
+      .string()
+      .email('Invalid email format')
+      .required('Email is required'),
+    password: yup
+      .string()
+      .min(8, 'Password must be at least 8 characters')
```

Comment:
I don't see a reason for checking password length in login form. The length should be checked when a new/fresh password is being set.